lib/topological_inventory/ansible_tower/operations/service_offering.rb

```diff
@@ -2,7 +2,8 @@
 require "topological_inventory/ansible_tower/operations/core/ansible_tower_client"
 require "topological_inventory/ansible_tower/operations/core/service_order_mixin"
 require "topological_inventory/ansible_tower/operations/core/topology_api_client"
-
+require "topological_inventory/ansible_tower/operations/applied_inventories/parser"
+require "pry-byebug"
```

Comment:
Don't forget to remove this

Reply:
I'm testing and writing specs now, but..yes, thanks, it happens to me :)